easydock/protonation.py

```diff
@@ -83,29 +79,26 @@ def read_smiles(fname):
             yield tuple(line.strip().split()[:2])
 
 
-def protonate_pkasolver(input_fname: str, output_fname: str, ncpu: int = 1):
+def protonate_pkasolver(input_smi: str, ncpu: int = 1, smi_size=1):
```

Comment:
`input_smi` is already not a SMILES and not of `str` type. Adjust please accordingly.
I suggest to rename it to `items`